doc/build_and_test.md

```
@@ -21,20 +21,48 @@ make tools
 | [charmbracelet/vhs](https://github.com/charmbracelet/vhs) | Write terminal GIFs as code for integration testing and demoing your CLI tools. |
 | [golangci/golangci-lint](https://github.com/golangci/golangci-lint) | Linters Runner for Go |
 | [mock/mockgen](https://github.com/uber-go/mock) | Mocking framework for the Go programming language |
 | [fe3dback/go-arch-lint](https://github.com/fe3dback/go-arch-lint) | GoLang architecture linter |
 
+`make tools` installs exactly the table above. It does not install
+[atago](https://github.com/nao1215/atago), the end-to-end runner, because the
+E2E suite runs the built `sqly` binary rather than linking a library; install it
+separately with the version CONTRIBUTING.md pins, which is the one CI uses:
 
+```shell
+$ go install github.com/nao1215/atago@v0.18.0
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)